spec/classes/opennebula_spec.rb

```diff
@@ -4,7 +4,10 @@
 
 describe 'one', :type => :class do
   context 'with default params as implicit hiera lookup' do
-    let (:facts) { {:osfamily => 'RedHat'} }
+    let (:facts) { {
```

Comment:
I'm a bit uncertain if that's what we want to do. We still deploy this on RHEL 6, so maybe we should just run the tests for both versions.